### ISSUE TITLE
Don't show any window on startup (or fix "start minimized")

### ISSUE THREAD

**Issue by hotsno** [34736384/genshin-fps-unlock#116]
I'd like a way to basically "forget" this app even exists when using it. I just want to be able to run the `.exe`, and then have Genshin launch as normal without the GUI showing up.

I think starting this app minimized is an okay compromise between being able to access the GUI when needed, and having the app stay out of your way. However, I have some issues with the "start minimized" option. The app still shows up in the Windows alt tab menu, which is quite annoying considering the purpose of the option is to have the app "stay out of your way"

![image](https://user-images.githubusercontent.com/71658949/218653275-fef84636-9a92-4ee2-a8ed-ebb323720a51.png)

So basically I propose either:
- Fixing the "start minimized" option to not show up in the alt tab menu, or
- Adding an option to completely hide the GUI and never have it show up.

Side note:
I noticed the "AutoClose" value in the config file, and tried enabling it since I thought maybe it does what I want, but I'm not sure what it does?

**Comment by 34736384** [comment]
fixed in new verion

**Comment by Trixlight** [comment]
This issue is back in https://github.com/34736384/genshin-fps-unlock/commit/07e9c9449847cde4a5347ec343b7ef69b07cc3e3

**Comment by hotsno** [comment]
Yeah it's been back for a while. In case it helps anyone, what I do is I just create a new desktop (`Win-Tab` > `New desktop`) and manually drag the app to that desktop every time.

This issue is quite annoying to me, as I really dislike having my `Alt-Tab` polluted, but I'm probably the minority lol.
